server/api/user.go

```diff
@@ -32,7 +32,7 @@ func (apiObj *API) initUser() {
 
 	apiObj.Users.GET("/", authMiddleware(), requireUserPermissions(), getUsers)
 	apiObj.Users.POST("/", authMiddleware(), requireUserPermissions(), createUser)
-	apiObj.Users.PUT("/", authMiddleware(), requireUserPermissions(), updateUser)
```

Comment:
Let's leave this for admins to use and add another endpoint `/patch_me` or something. Where you should check that only I can update my information.